src/niscope/metadata/config_addon.py

```diff
@@ -15,5 +15,22 @@
             'file_name': 'measurement_stats',
             'python_name': 'MeasurementStats'
         }
-    ]
+    ],
+    'repeated_capabilities': [
+        {
+            'attr_for_docs_example': 'channel_enabled',
```

Comment:
does the generator die if you put a non-existen attribute here? or does it use this string to look up additional info about the property?

Reply:
We don't use it to look anything up. We only use metadata from config.py, config_addon.py to build the rep cap examples. The only thing that happens if the attribute doesn't exist is the link to the attribute in the documentation won't work.